src/controller/python/test/test_scripts/commissioning_failure_test.py

```diff
@@ -106,7 +107,8 @@ async def main():
                       "Commissioning failure tests failed for simulated report failure on stage {}".format(testFailureStage))
 
     else:
-        for testFailureStage in range(3, 21):
+        # [kArmFailsafe, kConfigureTrustedTimeSource] (inclusive). TODO: https://github.com/project-chip/connectedhomeip/issues/36629
+        for testFailureStage in range(3, 20):
```

Comment:
This range change is odd. I also do not understand  the command

Reply:
The range being iterated is kArmFailsafe to kConfigureTrustedTimeSource (inclusive), before and after. The numbers changed because I removed kReadCommissioningInfo2. I added the todo because we really should be exposing these as named constants to Python somehow.